src/components/molecules/Modal.tsx

```diff
@@ -0,0 +1,46 @@
+import { PropsWithChildren, useEffect, useState } from 'react';
+import { Flex } from 'rebass';
+
+interface ModalProps {
+  isOpen: boolean;
+  onModalClose: VoidFunction;
+  onAfterOpen?: VoidFunction;
+  onAfterClose?: VoidFunction;
```

Comment:
```suggestion
  onClose: VoidFunction;
  onOpened?: VoidFunction;
  onClosed?: VoidFunction;
```

1. `onModalClose`: 이미 컴포넌트 이름만으로 `Modal` 이라는 맥락이 명확해서 이벤트리스너에서는 `Modal`이 빠져도 될 것 같사요!
2. `onAfterOpen`, `onAfterClose` : `After Open` 이라는 문법에 맞지 않는 표현보다는 과거형을 사용하면 의미도 맞고 알아보기도 쉬울 것 같아요!

Reply:
사용하던 라이브러리에서 사용하던 네이밍이 익숙해서 생각치도 못했네요! 변경했습니다~!

- https://github.com/Lubycon/DEVTI-front/pull/39/commits/b0c502fbf03f41639dc9325b9e9a5b978997fe47